src/commands/index.js

```diff
@@ -37,7 +36,7 @@ program.on('--help', () => {
   console.log('    $ refocus-collector --help');
   console.log('    $ refocus-collector start --collectorName=test ' +
     '--refocusUrl=https://refocus.abczyx.com --accessToken=eygduyguygijfdhkfjhkfdhg');
-  console.log('    $ refocus-collector stop --collectorName=test');
+  console.log('    $ refocus-collector1 stop --collectorName=test');
```

Comment:
are we explicitly changing refocus-collector to refocus-collector1?

Reply:
Will fix it. we are not changing the command name.